lib/utils/common-flags.ts

```diff
@@ -60,6 +60,7 @@ export const verbose: IBooleanFlag<boolean> = flags.boolean({
 export const yes: IBooleanFlag<boolean> = flags.boolean({
 	char: 'y',
 	description: 'answer "yes" to all questions (non interactive use)',
+	default: false,
```

Comment:
Might as well add `default: false` to the other boolean flags in this file (`verbose`, `force`).